app_dart/lib/src/service/bigquery.dart

```diff
@@ -35,14 +37,15 @@ select builder_name,
        array_agg(case when is_flaky = 1 then sha end IGNORE NULLS ORDER BY date DESC)[ordinal(1)] as recent_flaky_commit,
        array_agg(case when is_flaky = 1 then flaky_builds end IGNORE NULLS ORDER BY date DESC)[ordinal(1)] as flaky_build_of_recent_flaky_commit,
        sum(is_flaky)/count(*) as flaky_ratio
-from `flutter-dashboard.datasite.luci_prod_build_status`
-where date>=date_sub(current_date(), interval 14 day) and
-      date<=current_date() and
+from (select *, row_number() over (partition by builder_name order by time desc) as rank from `flutter-dashboard.datasite.luci_prod_build_status`)
+where date>=date_sub(current_date(), interval 30 day) and
       builder_name not like '%Drone' and
       repo='flutter' and
       branch='master' and
       pool = 'luci.flutter.prod' and
-      builder_name not like '%Beta%'
+      builder_name not like '%Beta%' and
```

Comment:
What about stable?

Reply:
Stable and dev do not run on `master` branch (we do not have any data under both `master` and `stable/dev`).

Reply:
Out of curiosity, why does beta run?

Reply:
@godofredoc may have more context.
BTW: here is the beta executions under `master`: go/flutter_beta_runs.

Reply:
Flakiness should be calculated only on master/main. Flakiness on release branches is not actionable as only approved cherry picks are allowed to land on those branches.

Reply:
This sounds like a bug in how we're saving the data (branch isn't being set correctly)

Reply:
Created: https://github.com/flutter/flutter/issues/93557